src/core/scheduler.ts

```diff
@@ -325,4 +323,16 @@ export class Scheduler extends EventEmitter {
     const errorPayload = await this.queue.forceCleanWorker(workerName);
     this.emit("cleanStuckWorker", workerName, errorPayload, delta);
   }
+
+  async watchIfPossible(key: string) {
+    if (typeof this.connection.redis.watch === "function") {
+      return this.connection.redis.watch(key);
+    }
+  }
+
+  async unwatchIfPossible() {
```

Comment:
I would recommend limiting visibility to `private` until there's an actual need for wider visibility.

Reply:
good idea!  https://github.com/actionhero/node-resque/pull/475/commits/ce306b8407cdb0e6ac1873e5250d167398aab52b